### ISSUE TITLE
Danobooru no longer fetches results

### ISSUE THREAD

**Issue by DeetrixReed** [Bionus/imgbrd-grabber#2426]
You are probably aware but...
Grabber does not show results from Danbooru anymore.  Been maybe a month since I last used, maybe I missed something major but I tired looking first.
Hopefully this can be working again, when you have time.  :)

![danbooru no results 2021 07](https://user-images.githubusercontent.com/77783894/126744854-9218c349-9b0f-48d3-80ac-626801a10242.png)

**Comment by yami-no-tusbas** [comment]
Working on nightly, you log my help, could be a cloudfalre issue, or anything else, just send the logs please.

**Comment by DeetrixReed** [comment]
Thank you for you work.
[main.log](https://github.com/Bionus/imgbrd-grabber/files/6872163/main.log)

**Comment by yami-no-tusbas** [comment]
Your login method is failing, look like you're using "HTTP Basic", you could try put it to "trought URL" at least it will work even without account. If you need your account try the "through url" method, since I have no account on this board I can help much more. But without account an using the "through url" method the board show me the result.

my log with : "through url" and no information set is 

```
[15:11:12.400][Info] [danbooru.donmai.us][Xml] Loading page https://danbooru.donmai.us/posts.xml?limit=20&page=1&tags=pink_pet_bottle
[15:11:12.728][Info] [danbooru.donmai.us][Xml] Receiving page https://danbooru.donmai.us/posts.xml?limit=20&page=1&tags=pink_pet_bottle
[15:11:12.875][Info] [danbooru.donmai.us][Xml] Parsed page https://danbooru.donmai.us/posts.xml?limit=20&page=1&tags=pink_pet_bottle: 20 images (20), 0 tags (336), -1 total (-1), -1 pages (-1)
```

and with http basic and no account set : 

```[15:11:37.520][Info] [danbooru.donmai.us][Xml] Loading page https://danbooru.donmai.us/posts.xml?limit=20&page=1&tags=pink_pet_bottle
[15:11:37.647][Info] [danbooru.donmai.us][Xml] Receiving page https://danbooru.donmai.us/posts.xml?limit=20&page=1&tags=pink_pet_bottle
[15:11:37.648][Warning] [danbooru.donmai.us][Xml] SessionLoader::AuthenticationFailure
[15:11:37.648][Warning] [danbooru.donmai.us] No valid source of the site returned result.```

**Comment by DeetrixReed** [comment]
Wow I wonder what happened, I had not touched anything since it last worked.
Well I worked at it and magically fixed it.  Tried all recommendations, and even when I removed all my account info I was still returning with nothing.  Restarted the program for good measure.  Thought to try a basic search with no field info.  It returned the front page!  I thought great, tried a one word search, it was gone again.  Re-Entered all my credentials, searched with no fields, it worked.  Searched for a few random tags, returned nothing.  So I was starting to narrow it down.  Then not sure what happened but I tried some other tags and they showed up.  I wanted to document the order of events as best I could remember, just in case it might help someone else.  Sorry I don't know what happened on the final step.  I had restarted it since I last asked but that seems to have helped.  Thanks a lot for your replies.  :)
